API/index.js

```diff
@@ -144,11 +142,15 @@ class API {
       /* -------------------------- Error Handling ---------------------------*/
       if (status === 429) {
         // On daily limit error, reject request, purge queue, and pause endpoint
-        if (headers['X-RateLimit-Type'] === 'daily') {
-          await throttle.rejectAllFromQueue(new ZACCLError({
-            message: 'Zoom is very busy right now. Please try this operation again tomorrow.',
-            code: ERROR_CODES.DAILY_LIMIT_ERROR,
-          }));
+        if (
+          headers['X-RateLimit-Type'] === THROTTLE_CONSTANTS.DAILY_LIMIT_HEADER
+        ) {
+          await throttle.rejectAllFromQueue(
+            new ZACCLError({
+              message: 'Zoom is very busy right now. Please try this operation again tomorrow.',
```

Comment:
This sounds like an error message for rate limiting. 
Perhaps we should go with 'The maximum daily call limit for this tool has been reached' or something to indicate daily tokens have been depleted

Reply:
This was Gabe's suggested error for this case, since the intended audience is the user not the programmers